### PR TITLE
Feature/db

### PR DESCRIPTION
**Pull request by EroAuditore**
# Database performance audit
:beginner: In this project we improved performance of next queries by adding indexes:
- [ ] SELECT COUNT(*) FROM visits where animal_id = 4;

- Before
animal_index_before

- After.
animal_index_after

- [ ] SELECT * FROM visits where vet_id = 2;

- Before
vet_index_after

- After.

vet_index_before

- [ ] SELECT * FROM owners where email = 'owner_18327@mail.com';

- Before

owners_index_before

- After
owners_index_after

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+CREATE INDEX animal_index ON visits (animal_id);
+CREATE INDEX vet_index ON visits (vet_id);
+CREATE INDEX owners_index ON owners (email);
```
Comment (DeliceLydia) [inline queries.sql:3]: ## Optional
- [ ] I suggest making the index on the case insensitive version of the email, 
As email addresses are not case sensitive, we might need to look for the emails in a case insensitive way. There is an option to reflect that in the index by using:
`CREATE INDEX ON owners ((lower(email)));`

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far! However, there are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
- [ ] The second query `SELECT * FROM visits where vet_id = 2;`

Before
bf

After
gt

If you check the execution time AFTER the improvements is higher than BEFORE the improvements which are not good, I suggest that you work on improving this so that the execution time after the improvements become lower than before the improvements 



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (EroAuditore) [reply queries.sql:3]: It was my mistake, I've changed the screenshot order, if you can see in line 4 in the before image is using the index, that one has the best time execution

Comment (DeliceLydia) [reply queries.sql]: @EroAuditore did you change the order of the screenshots?

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉

## Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**